### ISSUE TITLE
Default settings button is inconsistent

### ISSUE THREAD

**Issue by Vankog** [aleab/toastify#15]
When resetting to default (e.g. in Colors & Font), the current form values still show the old values until the window gets reopened. (e.g. color alpha)

https://github.com/aleab/toastify/issues/14#issuecomment-329990493:
> I actually pretty much never changed the "set to default" functionality since I forked the original project, so I don't really know what it does. I'll have a look at it soon.
> However, I'm pretty sure it resets everything, and not just the settings of the current tab.
> The fact that the values in the form don't get refreshed is probably caused by the fact that the window doesn't receive any notification of the changes – I'm not sure though.
> 
> I'll let you know.

----------

to be precise:

Setting some random settings all over the tabs:

![beforedefault001](https://user-images.githubusercontent.com/3804909/30515925-8820fc90-9b31-11e7-8ac0-b83e617a1048.png)
![beforedefault002](https://user-images.githubusercontent.com/3804909/30515927-88238da2-9b31-11e7-8082-3f03a441f218.png)
![beforedefault003](https://user-images.githubusercontent.com/3804909/30515924-881ffba6-9b31-11e7-8896-65ea46d08e5a.png)
![beforedefault004](https://user-images.githubusercontent.com/3804909/30515926-8821634c-9b31-11e7-966a-b2406193fe62.png)
-----------------------
Then hitting Default in the Toast tab at Color & Font:
Resets the internal values of Color & Font (see toast), but does not update the visible form values, yet:
![afterdefaultbeforeclose001](https://user-images.githubusercontent.com/3804909/30515938-1429aafc-9b32-11e7-96b1-262c5f8703fb.png)
Resets form values of the Toast -> General tab:
![afterdefaultbeforeclose002](https://user-images.githubusercontent.com/3804909/30515939-142be574-9b32-11e7-86f3-c16fc6ae7db9.png)
Does NOT reset key bindings:
![afterdefaultbeforeclose003](https://user-images.githubusercontent.com/3804909/30515940-142f3396-9b32-11e7-8798-de5f81b18548.png)
Resets the General tab settings (interestingly opting out of analytics):
![afterdefaultbeforeclose004](https://user-images.githubusercontent.com/3804909/30515941-143015cc-9b32-11e7-99a1-fae833dd3d81.png)
 
Closing and opening the settings loads the correct values into the form:
![afterdefaultreopen001](https://user-images.githubusercontent.com/3804909/30515953-588411c4-9b32-11e7-9a83-7d50ceda6002.png)

-------------
So in conclusion: there are some inconsistencies.
- it resets some things and not others
- it sets other defaults than the one from the first start after clean install (see analytics)
- it should update the values visible in the form consistently

and a feature wish:
I'd rather only reset just the currently viewed settings than all settings. For this I'd like to have another button. (e.g. "reset current settings" and "reset all settings")
But this is optional and has low priority for me.

**Comment by aleab** [comment]
**Perfect** issue report. 👍 
Will be fixed in v1.9.9.

**Comment by Vankog** [comment]
no pressure, it's merely a minor issue.

**Comment by aleab** [comment]
Fixed in the latest release.

**Comment by Vankog** [comment]
I think it works ell so far. With one exception. But I'll add another issue for this. ;-)
